resource-definitions/terraform-driver/dynamic-credentials/s3-dynamic-credentials.yaml

```diff
@@ -0,0 +1,29 @@
+# Connect to an EKS cluster using dynamic credentials defined via a Cloud Account
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: s3-dynamic-credentials
+entity:
+  name: s3-dynamic-credentials
+  type: s3
+  driver_type: humanitec/terraform
+  # The driver_account references a Cloud Account of type "aws-role"
+  # which needs to be configured for your Organization.
+  driver_account: aws-temp-credentials
+  driver_inputs:
+    values:
+      script: |-
```

Comment:
Might it make sense to highlight here that the script expects as input vars `ACCESS_KEY_ID`, `ACCESS_KEY_VALUE` and `SESSION_TOKEN`? For example:
```
provider "aws" {
  region     = var.REGION
  access_key = var.ACCESS_KEY_ID
  secret_key = var.ACCESS_KEY_VALUE
  token = var.SESSION_TOKEN
  default_tags {
    tags = {
      Humanitec = true
    }
  }
}
```

Reply:
That's great feedback, thanks! I added the provider block plus inline comments, and rearranged the order of the elements to better show the flow of credentials from `driver_account` -> `credentials_config` -> TF variables.